crates/red_knot/src/types.rs

```diff
@@ -509,8 +516,9 @@ mod tests {
         let store = TypeStore::default();
         let files = Files::default();
         let file_id = files.intern(Path::new("/foo"));
-        let id = store.add_function(file_id, "func");
+        let id = store.add_function(file_id, "func", vec![Type::Unknown]);
```

Comment:
used a random placeholder type; should I use something more specific?

Reply:
Yep for sure, in a test it's fine.